examples/container-service-extension/3.7-cse-step1.tf

```diff
@@ -0,0 +1,269 @@
+# Create an Organization and a VDC. VDC must be backed by NSX-T.
+
+resource "vcd_org" "cse_org" {
```

Comment:
Is this org required for CSE to work? If yes, may be worth explaining that more in the comment above.

Reply:
No, it is not. I have changed the comment to explain it better, let me know if it's clearer now!